docs/.vuepress/api.json

```diff
@@ -233,9 +233,12 @@
 						"path": "/api/titanium/ui/android",
 						"children": [
 							["titanium/ui/android/cardview", "CardView"],
+							["titanium/ui/android/collapsetoolbar", "CollapseToolbar"],
 							["titanium/ui/android/drawerlayout", "DrawerLayout"],
+							["titanium/ui/android/floatingactionbutton", "FloatingActionButton"],
 							["titanium/ui/android/progressindicator", "ProgressIndicator"],
-							["titanium/ui/android/searchview", "SearchView"]
+							["titanium/ui/android/searchview", "SearchView"],
+							["titanium/ui/android/snackbar", "Snackbar"]
```

Comment:
Are these generated or manually added? This JSON file should usually only be auto-generated.

Reply:
https://github.com/tidev/titanium-docs/tree/main/docs/api - api.json is automatically generated

this file had a manual change in 2022 too: https://github.com/tidev/titanium-docs/commit/ee2096201f5033eb9328abb1bb5c184651f348f6

Reply:
all changes https://github.com/tidev/titanium-docs/commits/main/docs/.vuepress/api.json are made by hand I would say

Reply:
Okay - so this is now auto-generated as well or manually as well? In case it's manually changed, we have to change the generator script instead

Reply:
only the api.json for the api docs is automatically generated (and still is). This is the menu structure for vuepress and by looking at all the older commits it was never auto generated

Reply:
Okay, then we're good! :)